src/lib/autocomplete/autocomplete.ts

```diff
@@ -50,6 +58,9 @@ export class MdAutocomplete implements AfterContentInit {
   /** Function that maps an option's control value to its display value in the trigger. */
   @Input() displayWith: (value: any) => string;
 
+  /** Event that is emitted whenever an option from the list is selected. */
+  @Output() select: EventEmitter<MdAutocompleteSelect> = new EventEmitter<MdAutocompleteSelect>();
```

Comment:
This is pretty much the only use for the `EventEmitter`: https://angular.io/docs/ts/latest/api/core/index/EventEmitter-class.html